server/graphql/general/src/queries/tests/master_lists.rs

```diff
@@ -67,15 +67,6 @@ mod graphql {
                   name
                   code
                   description
-                  lines {
-                      nodes {
-                          id
-                          item {
-                              id
-                          }
-                      }
-                      totalCount
-                  }
```

Comment:
What was the reason to remove this?

Reply:
That was a change in the master list line pagination PR, but missed it since action tests didn't fail 👀